06-building-a-program-start-to-finish/setting-up-continuous-integration-(build-server)/output/readme.md

```diff
@@ -0,0 +1,18 @@
+#### Output
+- **Using**: TravisCI
+- **Deliver**: a link to a successful TravisCI build of your `development` branch (it should include test runs)
+- **With**: *independently*
+
+#### Result
+- Integerated travis ci service with my repository
+- screen shots of travis build badges on my readMe.md file in the development branch
+
+### link to travis build result of my repository on travis-ci.org
+[link to travis build with 20 test passed](https://travis-ci.org/andela-uibrahim/inverted-index/builds/205248690)
```

Comment:
Check out verbose reporters, they'll help give more information on the tests execution.